### PR TITLE
Proposed Learning Outcomes

### PR DESCRIPTION
**Pull request by JonathanYiv**
Other content in the readings not included:
* typeof(x)
* Number(x)
* toString(x)
* str.length
* indexOf(), lastIndexOf(), search()
* replace(), toUpperCase(), toLowerCase()
* concat(), charAt(), charCodeAt()
* split()
* Double Not !! and Boolean(x)
* String Comparison Mechanisms

### CODE REVIEW

**Comment by JonathanYiv** [comment]
For the DevTools resource, the new docs are slightly different from the original docs you linked.
I tried to pick and choose the relevant docs that matched the material in the originals primarily.

File: JS101/fundamentals/fundamentals-2.md [hunk 1]
```diff
@@ -1,21 +1,43 @@
 ### Introduction
 There are a few extremely common types of data that you will encounter in Javascript, and these fundamentals lessons will give us a really strong foundation in all of them.  Before we start digging deep, however, [this article](http://javascript.info/types) will give you a quick overview of the most common ones.
 
+### Learning Outcomes
```
Comment (mindovermiles262) [inline JS101/fundamentals/fundamentals-2.md]: Can we turn these into dropdowns?

Comment (JonathanYiv) [reply JS101/fundamentals/fundamentals-2.md]: I can do that

**Comment by JonathanYiv** [comment]
Before I do that..
How do I avoid all these commit merges? They're messy.
Is it `git pull rebase`?
Or `git fetch git merge`?

**Comment by KevinMulhern** [comment]
I wouldn't rebase after merging in master, it rarely goes well 🙈 
You could squash the two latest master merges into one commit with an interactive rebase though.

**Comment by JonathanYiv** [comment]
I suppose I could just create a separate branch so this doesn't happen in the future

**Comment by mindovermiles262** [comment]
Agree with @KevinMulhern , rebasing isn't worth the hassle. It never goes well 🙈

**Comment by JonathanYiv** [comment]
OK, so branch for non-project changes and rebase only locally
for cleaner commits. 
I shall do so

File: JS101/fundamentals/rock-paper-scissors-project.md [hunk 2]
```diff
@@ -24,19 +24,21 @@ We're going to make a simple implementation of grade-school classic "rock paper
...
 
 ### Student Solutions
-Send us your solution so we can show others! Submit a link to the Github repo with your files in it by using any of the methods listed on the contributing page.  See the Google Homepage project for examples.
+Submit a solution with a pull request to this [file](https://github.com/TheOdinProject/javascript_curriculum/blob/master/JS101/fundamentals/rock-paper-scissors-project.md) on the Javascript Curriculum github repository. See the section on [Contributing](http://github.com/TheOdinProject/curriculum/blob/master/contributing.md) for how.
 
 - Add your solution below this line!
-- [James Moores's Gui solution](https://rawgit.com/James-N-M/rock-paper-scissors/master/index.html) 
+- [Jonathan Yiv's solution](https://github.com/JonathanYiv/rock-paper-scissors) - [View in browser](https://jonathanyiv.github.io/rock-paper-scissors/)
+- [James Moores's Gui solution](https://rawgit.com/James-N-M/rock-paper-scissors/master/index.html)
+>>>>>>> 05a8053f097048343a2953135f7eb9c02b05cdab
```
Comment (codyloyd) [inline JS101/fundamentals/rock-paper-scissors-project.md]: uh oh 👀

Comment (mindovermiles262) [reply JS101/fundamentals/rock-paper-scissors-project.md]: 😲

**Comment by JonathanYiv** [comment]
SHOULD be good now.

**Pull request merged**